packages/invoices/src/OutboundShipment/DetailView/api.ts

```diff
@@ -104,7 +104,7 @@ const invoiceToInput = (
     id: patch.id,
     // color: patch.color,
     comment: patch.comment,
-    status: patch.status as InvoiceNodeStatus,
+    status: patch.status as unknown as UpdateOutboundShipmentStatusInput,
```

Comment:
there's a little bit of hunchy funchy going on with the types, where `InvoiceNodeStatus` is all of the statuses but the type for this is something like `InsertOutboundShipmentStatusInput` which is a subset .. so that's really annoying